0_global_functions.R

```diff
@@ -46,9 +46,7 @@ set_col_types <- function(grouping_variables, fixed_col_types) {
 set_project_parameters <- function(file_path){
   cfg <- config::get(file = file_path)
 
-  if (!is.null(cfg$paths$data_location_ext)) {
-    data_location_ext <<- cfg$paths$data_location_ext
-  }
+  proj_data_location_ext <<- cfg$paths$data_location_ext
```

Comment:
This will write `NULL` if it doesn't exist in the config file?

Reply:
exactly

Reply:
```r
list()$test_nonexistant_name_element
# NULL
```

Reply:
let's ship it.